web/src/combinedStatusMessage.ts

```diff
@@ -0,0 +1,31 @@
+import { isZeroTime } from "./time"
+import { StatusItem } from "./Statusbar"
+
+const combinedStatusMessage = (resources: Array<StatusItem>): string => {
+  let buildingResources = resources.filter(
+    r => !isZeroTime(r.currentBuild.StartTime)
+  )
+
+  if (buildingResources.length > 0) {
+    return "Building: " + buildingResources[0].name
+  }
+
+  let containerCrashedResources = resources.filter(
+    r => r.podStatus === "CrashLoopBackOff"
```

Comment:
Note:
If we add an error on startup, we do see "Container crashed". But if we fix that error, we'll see "Building..." and a quick flash of "Container crashed" before the status message clears on success.